### ISSUE TITLE
gem display

### ISSUE THREAD

**Issue by hvds** [Tendsty/gooboo#55]
TLDR: I think topaz should show in gem tab from the point you know about them.

I unlocked events during the weather event; I think a note somewhere mentioned topaz, but they didn't show up in gem display. A day or two later I noticed that ruby purchases had acquired a new "gems" tab that offered "+200" to "topaz capacity". Thinking maybe the initial capacity was 0 and I was therefore losing out, I immediately saved up for the capacity increase, and was distressed to find that after buying it topaz still didn't show up.

It was only after the end of the event that I got to see the topaz display and could finally discover from the mouseover that they are not generated during events.

Possibly related: during the weather event, fishing rods showed as requiring an unknown currency to purchase. If you clicked on one, it would nonetheless pop up the "you are about to spend a valuable currency" confirmation dialog; I don't think it should pop that up if you can't buy the item anyway.

(BTW I love the spoiler-free patch notes: it's a brilliant idea, and one I haven't seen before. If the code is also spoiler-free I'd be happy to put together a PR.:)
